sys/shell/commands/sc_fib.c

```diff
@@ -71,7 +71,7 @@ int _fib_route_handler(int argc, char **argv)
     /* e.g. fibroute right now dont care about the adress/protocol family */
     if (argc == 1) {
         fib_print_routes();
-        return;
+        return 0;
```

Comment:
The changes in this file are rather unrelated to the PR description, but they only fix a warning regarding return values. 
So if there are no objections I would leave them inside and not open a new PR.